AutoGear.lua

```diff
@@ -2100,6 +2098,7 @@ SlashCmdList["AutoGear"] = function(msg)
 		if usingPawn then
 			pawnScaleName, pawnScaleLocalizedName = AutoGearGetPawnScaleName()
 		end
+		realSpec = tostring(realSpec)
```

Comment:
I looked at what's happening here, and it looks like Cataclysm has a different first return value for `GetTalentTabInfo`.  So, the `AutoGearDetectSpec` implementation for Cataclysm needs to select the second return value instead of the first for the spec name instead of calling `tostring` here, I think.  What do you think?

I say this without being able to test this (as I'm still not at my home PC with WoW available) because I did a github search for GetTalentTabInfo and found this line in the BigWigs code:

https://github.com/BigWigsMods/BigWigs/blob/53e8515d7c043e48714798cb16d9643eb818ab02/Core/BossPrototype_Classic.lua#L105

This indicates that BigWigs checks for Cataclysm Classic and selects a different return value because of that.  So, AutoGear should do the same.  Looks like the spec name string is the second return value instead of the first in Cataclysm Classic.

Reply:
I'm ok with that I was struggling on this one

Reply:
recent commit fixes the underlying problem

Reply:
I see this assignment is still here, and I don't think it's necessary since `realSpec` is already a string.

Reply:
dropped realspec to string